bap-builder/SysrootMode.go

```
@@ -0,0 +1,114 @@
+		return err
+	}
+	contextManager := bringauto_context.ContextManager{
+		ContextPath: contextPath,
+	}
+	logger := bringauto_log.GetLogger()
+	logger.Info("Checking Git Lfs directory consistency")
+	err = repo.CheckGitLfsConsistency(&contextManager, platformString, *cmdLine.ImageName)
+	if err != nil {
+		return err
+	}
+	packages, err := contextManager.GetAllPackagesStructs(platformString)
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)